src/lib/rule-finder.js

```diff
@@ -108,6 +108,6 @@ function RuleFinder(specifiedFile, options) {
   this.getUnusedRules = () => getSortedRules(unusedRules);
 }
 
-module.exports = function (specifiedFile, options = {}) {
+module.exports = function (specifiedFile, options) {
```

Comment:
I cannot simply push to this repo after using the git scalpel locally. So I would have reverted this with another separate commit when bumping the version number.